src/Workspaces/Core/Portable/Shared/Collections/IntervalTree`2.cs

```diff
@@ -35,20 +36,20 @@ public IntervalTree()
         {
         }
 
-        public IntervalTree(IIntervalIntrospector<T> introspector, IEnumerable<T> values)
+        public IntervalTree(in TIntrospector introspector, IEnumerable<T> values)
```

Comment:
oh, nevermind.  it's also used here, and constructors can't be generic.  pity.  

looking to see if this can be cleaned up.

Reply:
Resolved by making this a static `Create` method.